lib/wor/paginate/adapters/active_record.rb

```diff
@@ -15,10 +15,16 @@ def paginated_content
           @paginated_content ||= @content.offset(offset).limit(@limit)
         end
 
-        delegate :count, to: :paginated_content
+        def count
+          paginated_content.size
+        end
 
         def total_count
-          @content.count
+          content = @content.reorder(nil)
+          content_size = content.try(:size)
+          return content_size.count if content_size.is_a? Hash
```

Comment:
In `lib/wor/paginate/adapters/will_paginate.rb:24` you have another logic
```rb
# lib/wor/paginate/adapters/will_paginate.rb:24
return count.size if count.is_a? Hash
```
And here 
```rb
# lib/wor/paginate/adapters/active_record.rb
return content_size.count if content_size.is_a? Hash
```
It's ok?

Reply:
no